docs/docs/glossary.md

```diff
@@ -0,0 +1,70 @@
+- **unified** current EO cloud back-ends all have [a different API](http://r-spatial.org/2016/11/29/openeo.html), making EO data analysis hard to validate,difficult to reproduce, and back-ends difficult to compare in terms of capability and costs, or to combine in a joint analysis across back-ends. A unified API can resolve many of these problems.
+
+
+## Datasets
+
+CEOS ([CEOS OpenSearch Best Practice Document v1.2](http://ceos.org/ourwork/workinggroups/wgiss/access/opensearch/)) defines **Granules** and **Collections** as follows: "A ***granule*** is the finest granularity of data that can be independently managed. A granule usually matches the individual file of EO satellite data.", and "A ***collection*** is an aggregation of granules sharing the same product specification. A collection typically corresponds to the series of products derived from data acquired by a sensor on board a satellite and having the same mode of operation."
+
+The same document lists the synonyms used (by organisations) for:
+
+- **granule**: dataset (ISO 19115), dataset (ESA), granule (NASA), product (ESA, CNES), scene (JAXA)
+- **collection**: dataset series (ISO 19115), collection (CNES, NASA), dataset (JAXA), dataset series (ESA), product (JAXA)
+
+Here, we will use **granule** and **collection**.
+
+A *granule* will typically refer to a limited area and a single overpass leading to a very short observation period (seconds), or a temporal aggregation of such data as e.g. for 16-day MODIS composites.
+
+The open geospatial consortium published a document on [OGC OpenSearch Geo and Time Extensions](https://portal.opengeospatial.org/files/?artifact_id=56866).
+
+## Processes and Jobs
+
+The terms _process_, _process graph_ and _job_ have different meanings in the OpenEO API specification.
+
+A **process** is simply the description of an operation as provided by the back end, similar to a function definition in programming languages. 
+
+A **process graph** includes specific process calls, i.e. references to one or more processes including specific values for input arguments similar to a function call in programming. However, process graphs can chain multiple processes. In particular, arguments of processes in general can be again (recursive) process graphs, input datasets, or simple scalar or array values.
+
+A **job** brings one process graph to the back-end and organizes its execution, which may or may not induce costs. Jobs furthermore allow to run process graphs from different _data views_ (see section on [data views](views.md)). Views define at which resolution and extent we look at the data during processing and hence allow to try out process graphs on small subsets, or work interactively within web map applications. For more information about jobs and their evaluation types, see the section on [jobs](jobs.md).
+
+In this context OpenEO will:
```

Comment:
This chunk of text ("In this context OpenEO will" + the 4 bullet points after it) is out of context. I even think that such a feature description shouldn't be part of a glossary.

Reply:
This is indeed true. For now, I moved the text to the processes, where it belonged to originally. We might want to move the text to another part of the docs later as this is to detailed for a glossary.

Reply:
Moved to issue #26 for later handling.